content/_index.html

```diff
@@ -0,0 +1,495 @@
+---
+title: RAPIDS.ai | GPU accelerated data science
+description: homepage
+---
+
+<!-- Hero Shortcode -->
+{{< hero hero-text="GPU Accelerated Data Science" button-link="#quick-start" button-text="Quick Start">}}
+
+  <!-- Announcements Shortcode (0-3x)-->
+  <!-- UPDATE HERE -->
+  {{< announcements 
+    text-1="RAPIDS 23.02 Released!" link-1="https://docs.rapids.ai/install#selector"
+    text-2="NVIDIA GTC March 20-23!" link-2="https://www.nvidia.com/gtc/"  
+    text-3="Deprecation for Ubuntu 18.04 in v22.12" link-3="https://docs.rapids.ai/notices/rsn0024/" 
```

Comment:
From a "brand voice" perspective, I might like to see the exclamation marks removed.

Also, can we can point to "new support" instead of a deprecation in our top level announcement? I think that new Python version support is more relevant to our audience than dropping an old Ubuntu version.
```suggestion
    text-1="RAPIDS 23.02 Released" link-1="https://docs.rapids.ai/install#selector"
    text-2="NVIDIA GTC March 20-23" link-2="https://www.nvidia.com/gtc/"  
    text-3="Support for Python 3.10 in v23.02" link-3="https://docs.rapids.ai/notices/rsn0023/" 
```